### PR TITLE
Simple travis build parallelization between linting and testing

### PR DESCRIPTION
**Pull request by map34**
## Overview

Simple build parallelization between linting and testing

## Testing Instructions

 * Look at travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@ sudo: false
 # Specifies the linux distribution to run the CI on
 dist: trusty
 
+env:
+  global:
+    - CI_NODE_TOTAL=2
```
Comment (lsetiawan) [inline .travis.yml]: I haven't seen this option before. What does it do? Scale up?

File: backend/server/oceanhub/videos/views.py [hunk 2]
```diff
@@ -34,3 +34,4 @@ def kafkastream(consumer):
         logger.info(f'Offset: {msg.offset}')
         yield (b'--frame\r\n'
                b'Content-Type: image/jpeg\r\n\r\n' + msg.value + b'\r\n\r\n')
+"hehe"
```
Comment (lsetiawan) [inline backend/server/oceanhub/videos/views.py]: umm..... "hehe"?

Comment (map34) [reply backend/server/oceanhub/videos/views.py]: lol trying to fail this thing

File: .travis.yml (hunk 1 above)
Comment (map34) [reply .travis.yml]: Yeah I retracted from that and just use some ENV variable magic instead

**Pull request merged**